Sources/StreamChatUI/Common Views/ChatChannelCreateNewButton.swift

```diff
@@ -11,10 +11,11 @@ public typealias ChatChannelCreateNewButton = _ChatChannelCreateNewButton<NoExtr
 /// A Button subclass that should be used for creating new channels.
 open class _ChatChannelCreateNewButton<ExtraData: ExtraDataTypes>: Button, UIConfigProvider {
     override public func defaultAppearance() {
+        super.defaultAppearance()
         defaultIntrinsicContentSize = .init(width: 44, height: 44)
         setImage(uiConfig.images.newChat, for: .normal)
```

Comment:
There's no reason why the button should have this intrinsic content size, let's remove it completely.

Reply:
Keep in mind this will change the ChanneListVC snapshot tests, because this pushes the button of the navigation bar to the left a little. That is why I had put before a width of 60, so it would have the same style as before. But I don't know why we wouldn't change the default layout of the button in the Nav bar 🤔

Reply:
✅ Done